src/tox/_pytestplugin.py

```diff
@@ -47,17 +47,17 @@ def work_in_clean_dir(tmpdir):
         yield
 
 
-@pytest.fixture(name="newconfig")
-def create_new_config_file(tmpdir):
-    def create_new_config_file_(args, source=None, plugins=()):
+@pytest.fixture
+def newconfig(tmpdir):
+    def create_new_config_file_(args, source=None, plugins=(), load_entrypoints=False):
```

Comment:
Not sure if it should be added here as kwarg.  We could just use `False` below always, and add it if needed later?

Reply:
please revert this line, use the name to avoid non pep 8 function names

Reply:
`newconfig`?
Would `new_config` be better then?
(I think it is important for developers to goto the function easily, and ctags does not understand the redirection trhough `name` here)

Reply:
pycharm understands and it's up to the other tools to improve their smartness, not up to us to write badly formatted code. the name needs to be newconfig for backwards compatibility.

Reply:
Why is `newconfig` not PEP8 btw?

So you are saying "goto newconfig" in pycharm goes to this fixture?
You agree that it is not really trivial (but I can imagine pycharm has implemented it), since you a) need to be aware of pytest specialities and then b) parse any function decorators for it.

I'll revert it.

Reply:
I've also not exposed load_entrypoints in the fixture's kwargs here now.

> backwards compatibility

Isn't this just used with tox itself?

Reply:
it's a pytest plugin shipped with tox, e.g. plugins use it

Reply:
Then we should also default to the old behavior of loading entrypoints, no?

Reply:
Can you also my other questions, please?

Reply:
Also btw:

> it's up to the other tools to improve their smartness

https://github.com/blueyed/tox/blame/a2e4b804c150ac6303e0293f5e4517b2b4580c08/tests/conftest.py#L4

('# noqa` should be enough for all tools there)

Reply:
yes we probably should